### PR TITLE
DBSCAN subsampling

### PR DESCRIPTION
**Pull request by emmanuellujan**
Adding a new subsampling method.

### CODE REVIEW

File: src/Data/datatypes.jl [hunk 1]
```diff
@@ -20,6 +20,14 @@ Abstract type declaring the type of information that is unique to a particular a
 abstract type AtomicData <: Data end
 
 CFG_TYPE = Union{AtomsBase.FlexibleSystem,ConfigurationData}
+
+"""
+    get_values(v::SVector)
+
+Removes units from a position.
+"""
+get_values(v::SVector) = [v.data[1].val, v.data[2].val, v.data[3].val]
```
Comment (swyant) [inline src/Data/datatypes.jl:29]: Why not use the ustrip function?

**Review by swyant — APPROVED** [review]
I just had one unimportant comment, but otherwise this all looks good to me, if you want to merge this without Joanna's additional commits.

**Comment by emmanuellujan** [comment]
Hi! some comments below:
- The data.xyz file was deleted. This caused two of the tests to stop working. I think it is best to restore the file for now.
- Is it necessary to have the *pod files in github? I do not see them used in the subsampling scripts.
- About the subsampling example, I think we can push this example as is for now. However, there are functions in "subsampling_utils.jl" that will probably need to be moved to the "src" folder.

File: src/Data/datatypes.jl (hunk 1 above)
Comment (emmanuellujan) [reply src/Data/datatypes.jl]: :+1:

**Comment by emmanuellujan** [comment]
I will merge this PR so we can continue with other PRs. We can follow up later.

**Pull request merged**